src/notyf.options.ts

```diff
@@ -24,7 +24,7 @@ export interface INotyfNotificationOptions {
   type: string;
   className: string;
   duration: number;
-  icon: INotyfIcon | false;
+  icon: HTMLElement | INotyfIcon | false;
```

Comment:
I have doubts about this API. I like the formality — asking the input to be a well-formed HTML. However, this looks like too much for an icon markup, IMO. I'm thinking about passing HTML content, ala `icon: '<svg>whatever</svg>'` or `icon: '<i class="fa fa-whatever"></i>'` or even `icon: '❌'`. That'd be good enough to fulfill the request, and it's easy to compose. This would also save us from having to do expensive checks on HTML elements.

Reply:
But it will cause an XSS attack if the the user provides to his customers the ability to passing the input of `icon`

Reply:
Yeah, thanks for calling it out. That's actually a possibility now as you can pass custom HTML to the Notyf message. We discussed this in https://github.com/caroso1222/notyf/issues/72. I still think doing HTML sanitization is out of the scope of this project. If a dev allows users to put custom HTML anywhere in the page, without sanitizing (with Angular, React, or any framework), that's a security risk on the project, not on the library.